dash/include/dash/graph/internal/Graph.h

```diff
@@ -0,0 +1,758 @@
+#ifndef DASH__GRAPH__INTERNAL__GRAPH_H__INCLUDED
+#define DASH__GRAPH__INTERNAL__GRAPH_H__INCLUDED
+
+namespace dash {
+
+/**
```

Comment:
Since this is `internal/Graph.h` the classes in here should be put into the `dash::internal` namespace.